src/gozilla/storage/database/db.go

```diff
@@ -0,0 +1,482 @@
+package database
+
+import (
+	"fmt"
+	"time"
+
+	"gorm.io/gorm"
+	setting "aaditya-singh/gozilla/src/storage/config"

```

Comment:
Leaving this just for track that: Relative Path is linked to your system :)

Reply:
There are more modifications needed in the code. However, you're correct that it is for linking dependencies.